dev-docs/bidders/gjirafa.md

```diff
@@ -0,0 +1,30 @@
+prebid_1_0_supported : true
```

Comment:
Hi @agon-qurdina sorry for the late comment - can you please move the key-value pair `prebid_1_0_supported: true` inside the YAML header block?  This will break the site build.